### PR TITLE
Add dark mode versions for Course Pane's "Loading" animation and "No Results" picture

### PR DESCRIPTION
**Pull request by ChaseC99**
## Summary
Previously the loading animation was a video. Now it's a gif.
Also added a second gif for dark mode.

(Sorry the gifs are kind of quick)
light
dark

Additionally, added a dark mode image for "No Results"
image

## Test Plan
Loading Animation
- Search for a class in light mode
- Search for a class in dark mode
- Verify that the loading animation is shown correctly for both

No Result Picture
- Search with no valid queries in light mode
- Search with no valid queries in dark mode
- Verify that the "No results" picture shows correctly for both

## Issues
Relates to #155

### CODE REVIEW

File: client/src/components/CoursePane/CourseRenderPane.js [hunk 1]
```diff
@@ -182,9 +184,7 @@ class CourseRenderPane extends PureComponent {
         if (this.state.loading) {
             currentView = (
                 <div className={classes.loadingGifStyle}>
-                    <video autoPlay loop>
-                        <source src={loadingGif} type="video/mp4" />
-                    </video>
+                    <img src={AppStore.getDarkMode() ? darkModeLoadingGif : loadingGif} alt="Loading courses" />
```
Comment (ChaseC99) [inline client/src/components/CoursePane/CourseRenderPane.js:188]: Is there a better way to determine if the site is in dark mode?

**Comment by ChaseC99** [comment]
Additionally, added a dark mode image for "No Results"
image

File: client/src/components/CoursePane/CourseRenderPane.js (hunk 1 above)
Comment (devsdevsdevs) [reply client/src/components/CoursePane/CourseRenderPane.js:188]: I don't see a problem with this approach?

**Review by braedons — APPROVED** [review]
Code looks clean and straightforward. Testing was smooth. Seems fine to me!

**Pull request merged**